brands/shop/department_store.json

```diff
@@ -334,10 +334,27 @@
   "shop/department_store|Muji": {
     "tags": {
       "brand": "Muji",
+      "brand:en": "Muji",
+      "brand:jp": "無印良品",
```

Comment:
FYI, `ja` is the ISO&nbsp;639 language code for Japanese. Fixed in 6a5cd967c138e255a40e44562b811eec321e1c07.

Reply:
Thanks for picking up on that!